marketplaceItems/drawApp/draw_app.js

```diff
@@ -0,0 +1,504 @@
+//
+//  draw_ app.js
+//
+//  Created by Rebecca Stankus on 01/31/19
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+/* global  */
+
+(function() {
+    var DESKTOP_LINE_DISTANCE_FROM_AVATAR_M = 1;
+    
+    
+    var tablet = Tablet.getTablet('com.highfidelity.interface.tablet.system');
+    var button = tablet.addButton({
+        text: 'DRAW',
+        icon: Script.resolvePath('Assets/Icons/draw-i.svg'),
```

Comment:
`camelCase` for all folder names, please. i.e. rename `Assets` to `assets` and `Icons` to `icons`